src/skmatter/datasets/descr/csd-1000r.rst

```diff
@@ -15,33 +15,33 @@ Function Call
 Data Set Characteristics
 ------------------------
 
-    :Number of Instances: Each representation 100
+:Number of Instances: Each representation 100
 
-    :Number of Features: Each representation 100
+:Number of Features: Each representation 100
 
-    The representations were computed with [C1]_ using the hyperparameters:
+The representations were computed with [C1]_ using the hyperparameters:
 
-    :rascal hyperparameters:
+:rascal hyperparameters:
 
-    +---------------------------+------------+
```

Comment:
I am not sure if the indentation is correct here. It was inconsistent between the dataset files. If I read the docs correctly there should be no indentation.